app/domain/authentication/strategy.rb

```diff
@@ -206,6 +228,15 @@ def new_token(input)
       )
     end
 
+    def new_oidc_token(user_details)
```

Comment:
add todo comment explaining that this should be done in the new OidcTokenFactory and this factory should be used as a dependency in the new OidcStrategy

Reply:
Done